generators/app/index.js

```diff
@@ -69,6 +55,43 @@ Have fun!`);
   }
 };
 
+function installAndRun(appPath, options) {
+  const installer = options.installer;
+
+  const parcelPath = require.resolve(".bin/parcel");
+  const indexPath = path.join(appPath, "app/index.html");
+
+  return new Promise((resolve, reject) => {
```

Comment:
resolve and reject are never used. By the way you can make installAndRun async so that you do not have to instanciate a Promie

Reply:
I just got rid of the promise entirely. Once `installAndRun` kicks off parcel, the only thing that will stop the process is if the user kills it, so we'll never return normally from it.